ledgersmb/Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG branches="1.6,1.7,1.8,1.9,master"
```

Comment:
Very nice idea!

However, this drops 1.5 support in the container. Is that accidental?

Reply:
I seems to remember that we discussed removing it.

Reply:
Hmm. I may be inconsistent, but i prefer to keep 1.5 as long as feasible.